### PR TITLE
Updated selenium-standalone package to work with chrome 61

### PR DESCRIPTION
**Pull request by adoublebarrel**
### What is the context of this PR?
Travis builds started failing due to the release of Chrome 61 which we use in headless mode for the new_functional tests. This is a trial of the latest version of selenium standalone which should resolve this issue.

### How to review 
Builds should pass.

### CODE REVIEW

**Review by jonnyshaw89 — APPROVED** [review]

File: tests/new_functional/config.js [hunk 1]
```diff
@@ -44,48 +44,47 @@ let config = {
...
+      chaiAsPromised.transferPromiseness = browser.transferPromiseness;
+      browser.setViewportSize({
+        width: 1280,
+        height: 1024
+      });
+    }
   }
-
-};
+);
```
Comment (samiwel) [inline tests/new_functional/config.js:73]: What was the reason for having to extend the Phantom config. Was this required to get the chrome headless working or is it an unrelated change? Sorry just being nosey on a Runner PR since it would be nice to get the headless chrome running on Author as well :-)

Comment (adoublebarrel) [reply tests/new_functional/config.js:73]: Unrelated. I almost changed back to running Phantom to unblock us and realised that I'd broken the configuration in the move to Chrome headless. So this is really re-instating some of that config so we can easily switch back to Phantom if we need to.

Phantom doesn't cope with our new multiple surveys test though so we'd have to disable them if we did switch back.

**Review by yoganandkunche — APPROVED** [review]
